apollo-router/src/axum_factory/listeners.rs

```diff
@@ -208,13 +208,10 @@ macro_rules! handle_connection {
         let received_first_request = $received_first_request;
         tokio::pin!(connection);
         tokio::select! {
-            // the connection finished first
```

Comment:
the order doesn't matter unless `biased;` is used: https://docs.rs/tokio/latest/tokio/macro.select.html#fairness

did you change this only for readability or did you intend to change the priority of each branch?